configs/e2e/src/pages/page.ts

```diff
@@ -23,16 +23,17 @@ class Page {
         return this.getElements(`.mx-name-${widgetName}`);
     }
 
-    public get input() {
-        const inputWidget = this.getWidget("dataInput");
-        const inputElement = inputWidget.$("input");
-        inputElement.waitForDisplayed();
-        return inputElement;
-    }
-
     get header(): string {
         return this.getWidget("pageTitle1").getText();
     }
+
+    get modalDialog(): WebdriverIO.Element {
+        return $(".modal-dialog");
+    }
+
+    get modalDialogHeader(): WebdriverIO.Element {
+        return this.modalDialog.$("#mxui_widget_Window_0_caption");
+    }
 }
 
 const page = new Page();
```

Comment:
This can be done in one line: `export default new Page();`

Reply:
done